ciri/conf/config_args.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __COMMON_CONFIGURATION_ARGS__
+#define __COMMON_CONFIGURATION_ARGS__
+
+char* short_options = "u:t:";
+struct option long_options[] = {                                                                    
```

Comment:
This structure probably won't be available if `getopt_long`itself is not.

Reply:
Do you mean if `getopt_long` is not available?
Then, we need to do some checking like @jserv mentioned what PostgreSQL did, right?